src/app/store/networks/networks.slice.ts

```diff
@@ -0,0 +1,67 @@
+import { createEntityAdapter, createSlice, EntityId, PayloadAction } from '@reduxjs/toolkit';
+import { ChainID } from '@stacks/transactions';
+
+import { DEFAULT_MAINNET_SERVER, DEFAULT_TESTNET_SERVER } from '@shared/constants';
+
+export enum DefaultNetworkIds {
+  mainnet = 'mainnet',
+  testnet = 'testnet',
+  devnet = 'devnet',
+}
+
+export type DefaultNetworkNames = keyof typeof DefaultNetworkIds;
+
+export interface Network {
+  chainId: ChainID;
+  id: string;
+  name: string;
+  url: string;
+}
+
+const defaultCurrentNetworkId = DefaultNetworkIds.mainnet as EntityId;
+
+export const defaultCurrentNetwork = {
+  chainId: ChainID.Mainnet,
+  id: DefaultNetworkIds.mainnet,
+  name: 'Mainnet',
+  url: DEFAULT_MAINNET_SERVER,
+} as const;
+
+export const defaultNetworks = {
+  [DefaultNetworkIds.mainnet]: defaultCurrentNetwork,
+  [DefaultNetworkIds.testnet]: {
+    chainId: ChainID.Testnet,
+    id: DefaultNetworkIds.testnet,
+    name: 'Testnet',
+    url: DEFAULT_TESTNET_SERVER,
+  },
+  [DefaultNetworkIds.devnet]: {
+    chainId: ChainID.Testnet,
+    id: DefaultNetworkIds.devnet,
+    name: 'Devnet',
+    url: 'http://localhost:3999',
+  },
+};
```

Comment:
These could better live elsewhere now not used in reducer. I would keep in constants file imo, that's where all the hard coded value are.